pkg/cli/powertools/kubecli/client.go

```diff
@@ -207,3 +212,7 @@ func (c *Client) GetObject(ctx context.Context, options ObjectOptions) (*unstruc
 	}
 	return u, nil
 }
+
+func (c *Client) Update(ctx context.Context, u *unstructured.Unstructured, opts ...client.UpdateOption) error {
+	return c.Client.Update(ctx, u, opts...)
+}
```

Comment:
If we are just passing this through do we need to wrap the `c.Client.Update` func?